# variance homogeneity test


## Case 1 two tailed test

After a teaching reform experiment, 10 and 9 papers were randomly selected from the classes with two different teaching methods, and the following results were obtained:

Test class: 85, 76, 83, 93, 78, 75, 80, 79, 90, 88

Comparison class: 75, 86, 96, 90, 62, 83, 95, 70, 58

To compare the effect of the test, first check whether the variance is homogeneous, so as to select the appropriate test method (α = 0.05)



H0 : ${\sigma_1}^2$ $=$ ${\sigma_2}^2$

${S_1}^2$ = 194.5, ${S_2}^2$ = 37.8

F = $\frac{{S_1}^2}{{S_2}^2}$ = $\frac{{194.5}}{{37.8}}$

In [2]:
import math
F = 194.5/37.8
print(F)

5.145502645502646



${F_{0.05/2}}(8,9)$ = 4.10

F=5.15 > 4.10

Therefore, there is a significant difference in the variance between the two classes



## Case 2 two tailed test

41 boys and 31 girls were randomly selected for the test. The standard deviation of male and female samples was 7 and 6 respectively. Are there any significant differences in the test results of boys and girls? (alpha =0.05)



H0 : ${\sigma_1}^2$ $=$ ${\sigma_2}^2$

${S_1}^2$ = 49, ${S_2}^2$ = 36

F = $\frac{{S_1}^2}{{S_2}^2}$ = $\frac{{49}}{{36}}$

In [3]:
import math
F = 49/36
print(F)

1.3611111111111112



${F_{0.05/2}}(40,30)$ = 2.01

F=1.36 < 2.01

Therefore, there is no significant difference in the variance of male and female performance
